Portal/vip-portal/src/main/resources/vm/wrapper.vm

```diff
@@ -73,6 +73,35 @@ fi
 \rm lcg-log 
 }
 
+function checkBosh {
+  local BOSH_CVMFS_PATH=$1
+  #by default, use CVMFS bosh
+  ${BOSH_CVMFS_PATH}/bosh create foo.sh 
+  if [ $? != 0 ]
+  then
+    info "CVMFS bosh in ${BOSH_CVMFS_PATH} not working, checking for a local version"
+    bosh create foo.sh 
+    if [ $? != 0 ]
+    then
+        info "bosh is not found in PATH or it is does not work fine, searching for another local version"
+        local HOMEBOSH=`find $HOME -name bosh`
+        if [ -z "$HOMEBOSH" ]
+        then
+            info "bosh not found, trying to install it"
+            pip install --trusted-host pypi.org --trusted-host pypi.python.org --trusted-host files.pythonhosted.org boutiques --prefix $PWD || error "pip install boutiques failed"
```

Comment:
In case of error (if there is not any working bosh), I think it will still try to execute it and fail ?